### ISSUE TITLE
Convert VAMC system service descriptions to "Rich text Limited" 

### ISSUE THREAD

**Issue by kevwalsh** [department-of-veterans-affairs/va.gov-cms#6828]
## User Story or Problem Statement

The audit in #6768 revealed opportunities for improvements to make the content more acessible and usable. 

When we run the script from #6456 to strip out `"<h3>Care we provide at [health care system]</h3>"` we have some other hardening work to do
- [ ] Convert the rich text to rich text limited

19 `<h2>`s  - these cause accessibility and usability problems
5822 `<h3>`s - (mostly "Care we provide at [health care system]”)
53 `<h4>`s 
4 `<h5>`s 

<details><summary>Not in scope of this</summary>

- [ ] Convert existing headings to `<strong>` OR just let editors discover their headings will disappear upon next save

</details>

## Acceptance Criteria

- [ ] Convert the System service description field to "Rich text limited" only and run a node save on all VAMC system health services to pick up the change.
- [ ] Validate that running updating the field format to only allow rich text limited, then resaving all nodes, will have desired effect
- [ ] Prepare PR for merging but do not merge until we're ready to trigger the runbook

## Implementation steps
<optional>

## Design principles

Veteran-centered
- [ ] `Single source of truth`: Increase reliability and consistency of content on VA.gov by providing a single source of truth.
- [x] `Accessible, plain language`: Provide guardrails and guidelines to ensure content quality.
- [ ] `Purposely structured content`: Ensure Content API can deliver content whose meaning matches its structure.
- [x] `Content lifecycle governance`: Produce tools, processes and policies to maintain content quality throughout its lifecycle.

Editor-centered
- [ ] `Purpose-driven`: Create an opportunity to involve the editor community in VA’s mission and content strategy goals.
- [ ] `Efficient`: Remove distractions and create clear, straightforward paths to get the job done.
- [ ] `Approachable`: Offer friendly guidance over authoritative instruction.
- [ ] `Consistent`: Reduce user’s mental load by allowing them to fall back on pattern recognition to complete tasks.
- [ ] `Empowering`: Provide clear information to help editors make decisions about their work.

### CMS Team

Please leave only the team that will do this work selected. If you're not sure, it's fine to leave both selected.

- [ ] `Platform CMS Team`
- [x] `Sitewide CMS Team`

**Comment by swirtSJW** [comment]
This issue did not have an associated PR.  I don't think this got done.
